backends/xnnpack/test/recipes/test_xnnpack_recipes.py

```diff
@@ -70,12 +78,8 @@ def test_int8_dynamic_quant_recipe(self) -> None:
                         example_inputs=example_inputs,
                         export_recipe=export_recipe,
                     )
-                    self.assertTrue(
-                        torch.allclose(
-                            session.run_method("forward", example_inputs[0])[0],
-                            m_eager(*example_inputs[0]),
-                            atol=1e-1,
-                        )
+                    self._compare_eager_quantized_model_outputs(
+                        session, example_inputs, 1e-1
```

Comment:
atol? Why is this so high as a general default?
```suggestion
                        session, example_inputs, atol=1e-1
```